tests/test_resolve.py

```diff
@@ -99,9 +100,12 @@ def test_iopro_mkl(self):
         ])
 
     def test_mkl(self):
-        a = r.install(['mkl 11*', MatchSpec(track_features='mkl')])
-        b = r.install(['mkl'])
-        assert a == b
+        SubdirData._cache_.clear()
+        Channel._cache_.clear()
```

Comment:
Should this be part of a `setUp` method for all tests?

Reply:
This is still an experiment, and apparently, it does not help this test to pass.

Reply:
Update: this test can pass